services/tests/test_decoration.py

```diff
@@ -376,6 +376,83 @@ def test_get_decoration_type_not_pr_plan(self, dbsession, mocker, enriched_pull)
         assert decoration_details.reason == "Org not on PR plan"
         assert decoration_details.should_attempt_author_auto_activation is False
 
+    def test_get_decoration_type_users_plan(self, dbsession, mocker, enriched_pull):
```

Comment:
This is a temporary test, this was the least effort way to replicate the logic for enriched pulls